app.py

```diff
@@ -54,34 +55,73 @@ def update_category(id_):
   category=Category.query.filter_by(id=id_).first()
 
   if category is None:
-    return jsonify(error=404, text="Category does not exist"), 404
+    return jsonify(error=404, text=strings.category_not_found), 404
 
   category = update_or_create_category(category=category, data=request.form)
   db.session.add(category)
   db.session.commit()
 
   return  jsonify(category.serialize())
 
-@app.route("/criteria")
+@app.route("/criteria", methods=["GET"])
 def get_criteria():
 	try:
 		criteria=Criterion.query.all()
 		return  jsonify([criterion.serialize() for criterion in criteria])
 	except Exception as e:
 		return(str(e))
 
-@app.route("/criteria/<id_>")
+@app.route("/criteria", methods=["POST"])
+@cross_origin(headers=["Content-Type", "Authorization"])
+@requires_auth
+def create_criterion():
+  try:
+    data = request.get_json()
+
+    category = Category.query.filter_by(id=data.get('category_id')).first()
+    if category is None:
+      return jsonify(text=strings.category_not_found), 404
+  
+    criterion = Criterion.create(**data)
+    return jsonify(criterion.serialize()), 201
+
+  except Exception as e:
+    return jsonify(text=str(e)), 500
```

Comment:
The error handling in this PR is kind of all over the place, but hopefully we'll make things consistent in #57 :)